cli/src/main/java/com/devonfw/tools/ide/tool/eclipse/Eclipse.java

```diff
@@ -84,21 +83,11 @@ public void installPlugin(ToolPluginDescriptor plugin, Step step) {
         }
       }
     }
-    log(IdeLogLevel.DEBUG, result.getOut());
-    log(IdeLogLevel.ERROR, result.getErr());
+    result.log(IdeLogLevel.DEBUG, context);
+    result.log(IdeLogLevel.ERROR, context);
```

Comment:
before, `out` and `error` have been logged on different log-levels.
Now, you IMHO log both on both levels causing duplications.